doc/modules/ROOT/pages/nrepl-api/ops.adoc

```diff
@@ -4,10 +4,28 @@ This file is _generated_ by #'cider.nrepl.impl.docs/-main
 ////
 = Supported nREPL operations
 
-[small]#generated from a verbose 'describe' response (cider-nrepl v0.25.3)#
+[small]#generated from a verbose 'describe' response (cider-nrepl v0.25.9)#
 
 == Operations
 
+=== `add-middleware`
```

Comment:
Seems we forgot to exclude the new built-in nREPL ops from the docs generator.

Reply:
`add-middleware`, `lookup`, and `swap-middleware` should all be excluded by the docs generator?

Reply:
Yep.